### DISCUSSION TITLE
There are two different input, how should I explain?

### DISCUSSION THREAD

**Discussion by wangmengyun1998** [shap/shap#2953] [Q&A]
model is cnn-lstm，The input form of the model is model (xs, con). xs is the input of cnn, and the dimension is 5. After dimension reduction,xs and con are input into lstm together

**Comment by jainraj** [comment]
Can you maybe give a block diagram to help you better? 

General idea is that SHAP values do not depend on number of inputs / model architecture etc. [SHAP value estimation algorithms will]. For starting, I would suggest Gradient Explainer and work from there!
